webrender/src/tiling.rs

```diff
@@ -27,6 +27,8 @@ use texture_allocator::GuillotineAllocator;
 use webrender_api::{DevicePixel, FontRenderMode};
 
 const MIN_TARGET_SIZE: u32 = 2048;
+const SOLID_SOLID: i32 = ((BorderStyle::Solid as i32) << 8) | ((BorderStyle::Solid as i32) << 16);
```

Comment:
`STYLE_SOLID`?

Reply:
I called it `SOLID_SOLID` because it matches both `style0` and `style1` being `BorderStyle::Solid`

Reply:
oh, I see now. Yes, `STYLE_SOLID_SOLID` sounds good to me